com.unity.netcode.gameobjects/Runtime/Transports/UTP/SecureAccessor.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.IO;
+using UnityEngine;
+
+namespace Unity.Netcode.Transports.UTP
+{
+    public class SecureAccessor : MonoBehaviour
+    {
+        private void Awake()
+        {
+            var serverSecrets = new ServerSecrets();
+
+            try
+            {
+                serverSecrets.ServerCertificate = ServerCertificate;
+            }
+            catch (Exception exception)
+            {
+                Debug.Log(exception);
+            }
+
+            try
+            {
+                serverSecrets.ServerPrivate = ServerPrivate;
+            }
+            catch (Exception exception)
+            {
+                Debug.Log(exception);
+            }
+
+            var clientSecrets = new ClientSecrets();
+            try
+            {
+                clientSecrets.ClientCertificate = ClientCA;
+            }
+            catch (Exception exception)
+            {
+                Debug.Log(exception);
+            }
+
+            try
+            {
+                clientSecrets.ServerCommonName = ServerCommonName;
+            }
+            catch (Exception exception)
+            {
+                Debug.Log(exception);
+            }
+
+            GetComponent<UnityTransport>().SetServerSecrets(serverSecrets);
+            GetComponent<UnityTransport>().SetClientSecrets(clientSecrets);
+        }
+
+        [Tooltip("Hostname")]
+        [SerializeField]
+        private string m_ServerCommonNameString = "localhost";
+        public string ServerCommonNameString
+        {
+            get => m_ServerCommonNameString;
+            set => m_ServerCommonNameString = value;
+        }
+        private string m_ServerCommonName;
+        public string ServerCommonName
+        {
+            get => m_ServerCommonNameString;
+        }
+        [Tooltip("Client CA filepath")]
+        [SerializeField]
+        private string m_ClientCAFilePath = "Assets/Secure/myGameClientCA.pem";
+        public string ClientCAFilePath
+        {
+            get => m_ClientCAFilePath;
+            set => m_ClientCAFilePath = value;
+        }
+
+        [Tooltip("Client CA Override. Certificate content, for platforms that lack file access (WebGL)")]
```

Comment:
Not sure we should be alluding to using CA certificates directly in WebGL. If using WebSockets, I expect most (if not all) users will be using a certificate from a known provider, in which case they wouldn't be setting up a CA certificate in UTP at all.

Reply:
I'll improve the tooltip comment.